llvm/tools/llvm-objdump/llvm-objdump.cpp

```diff
@@ -1366,8 +1375,9 @@ static SymbolInfoTy createDummySymbolInfo(const ObjectFile &Obj,
                                           uint8_t Type) {
   if (Obj.isXCOFF() && (SymbolDescription || TracebackTable))
     return SymbolInfoTy(std::nullopt, Addr, Name, std::nullopt, false);
-  else
-    return SymbolInfoTy(Addr, Name, Type);
+  if (Obj.isWasm())
+    return SymbolInfoTy(Addr, Name, wasm::WASM_SYMBOL_TYPE_SECTION);
```

Comment:
Why create a section symbol here where creating a dummy symbol?

Reply:
It doesn't actually matter what the symbol kind is, it just has to be distinguishable from a regular function symbol in WebAssemblyDisassembler.cpp. Its address is the beginning of the section so it seemed reasonable to call it a section symbol, but we could come up with something else; perhaps a dummy symbol type or something else on the SymbolInfoTy object.

Reply:
This function takes `Type` argument.. can we not use that?  Why ignore it?  (Full disclosure, I don't know what this function is for).